examples/tca9548a_simpletest.py

```diff
@@ -1,25 +1,19 @@
-# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
+# SPDX-FileCopyrightText: 2021 Carter Nelson for Adafruit Industries
 # SPDX-License-Identifier: MIT
 
-# This example shows using two TSL2491 light sensors attached to TCA9548A channels 0 and 1.
-# Use with other I2C sensors would be similar.
-import time
+# This example shows using sensors attached to TCA9548A channels 1 and 5.
```

Comment:
Same here.

Reply:
Solved in https://github.com/adafruit/Adafruit_CircuitPython_TCA9548A/pull/35/commits/8d66db73557f009d38d79dfca13f7e8d0e8e88ff
Thanks